### PR TITLE
Autoyast: Added bridge configuration example.

### PR DESCRIPTION
**Pull request by teclator**

### CODE REVIEW

File: xml/ay_bigfile.xml [hunk 1]
```diff
@@ -5241,6 +5241,37 @@ config:type="boolean"&gt;true&lt;/keep_install_network&gt;</screen>
 &lt;/networking&gt;</screen>
    </example>
 
+   <example>
+     <title>Bridge interface configuration</title>
```
Comment (sknorr) [inline xml/ay_bigfile.xml]: Please use title style: `Bridge Interface Configuration`

File: xml/ay_bigfile.xml [hunk 2]
```diff
@@ -5241,6 +5241,37 @@ config:type="boolean"&gt;true&lt;/keep_install_network&gt;</screen>
 &lt;/networking&gt;</screen>
    </example>
 
+   <example>
```
Comment (sknorr) [inline xml/ay_bigfile.xml]: It is generally a good idea to add `xml:id`s to formal elements that have a title (procedure, example, figure, ...)

**Comment by teclator** [comment]
@sknorr by now we will just add the example for autoyast, let see if deserve to add something more in the future, updated the PR according to your comments

**Comment by sknorr** [comment]
Thanks!

**Pull request merged**